opendbc/car/honda/interface.py

```diff
@@ -173,7 +183,7 @@ def _get_params(ret: structs.CarParams, candidate, fingerprint, car_fw, alpha_lo
       else:
         ret.lateralParams.torqueBP, ret.lateralParams.torqueV = [[0, 4096], [0, 4096]]  # TODO: determine if there is a dead zone at the top end
 
-    elif candidate == CAR.HONDA_PILOT:
+    elif candidate in (CAR.HONDA_PILOT, CAR.HONDA_PILOT_4G):
```

Comment:
let's try to use torque controller

Reply:
Do you have documentation on this for me to read to make this change?

Reply:
Oh, I thought we already used it for the radarless cars, but that's not true.